service/topology_coordinator.cc

```diff
@@ -3048,6 +3059,11 @@ future<> topology_coordinator::refresh_tablet_load_stats() {
         // the average tablet size by dividing total size by tablet count.
         table_load_stats.size_in_bytes /= table_total_replicas;
     }
+
+    if (table_load_stats_invalid) {
+        stats.tables.clear();
```

Comment:
Why are they invalid if one node is down? The rest can still be balanced.

Reply:
They are invalid because of the way we calculate the stats. We divide total size by RF. We could change it, but it's out of scope.